common/src/main/java/org/apache/uniffle/common/util/ThreadUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.uniffle.common.util;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import java.util.concurrent.ThreadFactory;
+
+/**
+ * Provide a general method to create a thread factory to make the code more standardized
+ */
+public class ThreadUtils {
+
+  public static ThreadFactory getThreadFactory(String factoryName) {
+    return new ThreadFactoryBuilder().setDaemon(true).setNameFormat(factoryName).build();
+  }
+}
```

Comment:
Could you add new line? Otherwise, there will be a format warning.

Reply:
done

Reply:
Not 31th line. Could you add a new line at the end of file?

Reply:
done